pkg/controllers/consolidation/controller.go

```diff
@@ -192,6 +195,18 @@ func (c *Controller) candidateNodes(ctx context.Context) ([]candidateNode, error
 			return true
 		}
 
+		// skip any nodes that we can't determine the capacity type or the topology zone for
+		ct, ok := n.Node.Labels[v1alpha5.LabelCapacityType]
+		if !ok {
+			logging.FromContext(ctx).Errorf("Getting node capacity type, %v", err)
```

Comment:
This has the potential to be quite noisy in the logs. Consider a set of nodes w/o capacity type. It will spew error logs on every consolidation loop. There are lots of other cases where we skip the node, is there a reason to log here, but not there?

Reply:
Maybe we can write up some guidelines together. This comes up fairly often.